### ISSUE TITLE
Find buttons' images don't render when editing

### ISSUE THREAD

**Issue by u9g** [retypeapp/retype#318]
On the latest npm version of retype, here's what the find bar looks like when editing:
![rZEcr7B](https://user-images.githubusercontent.com/43508353/162604584-16528911-e19d-4abc-9aca-417e8ff07a58.png)

**Comment by geoffreymcgill** [comment]
Thank you. I was was able to reproduce the issue. Looks like it has something to do websites that are hosted within a sub-folder, so must be a path issue with resources not being loaded properly. 

```
url: example.com/docs <-- image error
url: example.com      <-- image works
```

We will try and track down what is causing the problem.

**Comment by geoffreymcgill** [comment]
I forgot to post an update, but this defect has been fixed in the Retype v2.3.0 release. 

Thanks for reporting!
